ideascube/mediacenter/management/commands/clean.py

```diff
@@ -37,14 +38,28 @@ def add_arguments(self, parser):
             '--type', action='append', choices=document_types,
             help='The type of media to remove, e.g "--types=image". Can be '
                  'specified multiple times to clean multiple types.')
+        clean_media.add_argument(
+            '--created-before', type=date_argument,
+            help='Only remove media created before this date.')
+        clean_media.add_argument(
+            '--created-after', type=date_argument,
+            help='Only remove media created after this date.')
```

Comment:
To be honest, the fact that it will match medias created exactly at `datetime(YYYY, MM, DD, 0, 0, 0)` when specifying `--created-after=YYYY-MM-DD` is a bit of an easter egg.

I mean, statistically, no media will ever be created exactly at midnight, **0 minutes and 0 seconds**.

But better safe than sorry, I added a test as requested. 😃